src/model/modifier/DraftModifier.js

```diff
@@ -211,6 +212,18 @@ var DraftModifier = {
     );
   },
 
+  setBlockData: function(
+    contentState: ContentState,
+    selectionState: SelectionState,
+    blockData: DraftBlockType
```

Comment:
Surely this should be a `Map` or another collection of some kind, rather than a `DraftBlockType`?